services/dy-static-file-server/Makefile

```
@@ -24,6 +24,10 @@ export COMPOSE_INPUT_DIR  := ./validation/input
 export COMPOSE_OUTPUT_DIR := .tmp/output
 
 APP_NAME := dy-static-file-server
+LOCAL_REGISTRY_HOST = registry:5000
```

```suggestion
LOCAL_REGISTRY_HOST ?= registry:5000
```

this way you can override it from outside